frontend_vue/src/components/base/BaseButton.vue

```diff
@@ -1,11 +1,22 @@
 <template>
-  <button
+  <component
+    :is="link ? 'a' : 'button'"
```

Comment:
this seems a little weird no? what's the benefit of overloading the button to be a link / button? I think we're drifting a little too far into modularisation land here

Reply:
like... if we're just using it in one place why not just have a standalone link with some base button styling?

Reply:
The idea is that we gonna have some places where the links are styled like buttons. 

I don't see the advantage of copy/pasting the BaseButton code into BaseLinkButton.
It can eventually lead to design inconsistencies, or having to fix both components every time make a change in one of them.
I think it's easier to have one line of code that takes care of this instead.

If you think it's not clear and ppl might not find it easily, I can also create a BaseLink which is simply a BaseButton component with the link prop

Reply:
So this one "Find out" doesn't have any styling right? it's just text? Why not just use an anchor element?

Reply:
it's text with an icon, and it takes the same style as the text button.
So far this type of link is used only here. 

If you think it makes it all complicated to follow, I might just add an anchor and style it here. If we'll ever need more of these button-styled-link in the future, we'll think about another component then

Reply:
yeah i think we might be overcomplicating the button here where an simple Anchor works?

Reply:
Even an anchor component seems like a lot? like... an anchor class maybe?